lib/msal-react/src/hooks/useMsalAuthentication.ts

```diff
@@ -48,10 +74,60 @@ export function useMsalAuthentication(
                 logger.verbose("useMsalAuthentication - Calling ssoSilent");
                 return instance.ssoSilent(loginRequest as SsoSilentRequest);
             default:
-                throw "Invalid interaction type provided.";
+                throw ReactAuthError.createInvalidInteractionTypeError();
         }
     }, [instance, interactionType, authenticationRequest, logger]);
 
+    const acquireToken = useCallback(async (callbackInteractionType?: InteractionType, callbackRequest?: SilentRequest): Promise<AuthenticationResult|null> => {
+        const fallbackInteractionType = callbackInteractionType || interactionType;
+
+        let tokenRequest: SilentRequest;
+
+        if (callbackRequest) {
+            tokenRequest = {
+                ...callbackRequest
+            };
+        } else if (authenticationRequest) {
+            tokenRequest = {
+                ...authenticationRequest,
+                scopes: authenticationRequest.scopes || OIDC_DEFAULT_SCOPES
```

Comment:
Is this only needed because `scopes` are required for login requests?

Reply:
If so, that is something we've been meaning to address: #4312

Reply:
It's because `acquireTokenSilent` requires scopes and they may not have been provided since the current `useMsalAuthentication` implementation only calls login

Reply:
Ah, yes makes sense.